services/backend/src/routes/connectivity.js

```diff
@@ -192,5 +193,44 @@ module.exports = (errorHandler, connectivityService) => {
       });
   }));
 
+  /**
+   * @apiGroup s_pa_connectivity
+   * @api {get} /connectivity/se/layer SELayer
+   * @apiName SELayer
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Layer of a estrategic ecosystem in a given area
+   *
+   * Value calculated for 2020
+   *
+   * @apiParam (Query params) {String|Number} areaType area type
```

Comment:
Check again the param types of areaType and areaId

Reply:
Done